pkgs/tools/security/gnupg/23.nix

```
@@ -0,0 +1,103 @@
+{ fetchurl, fetchpatch, lib, stdenv, pkg-config, libgcrypt, libassuan, libksba
+, libgpg-error, libiconv, npth, gettext, texinfo, buildPackages
+
+# Each of the dependencies below are optional.
+# Gnupg can be built without them at the cost of reduced functionality.
+, guiSupport ? true, enableMinimal ? false
+, adns ? null , bzip2 ? null , gnutls ? null , libusb1 ? null , openldap ? null
+, tpm2-tss ? null
+, pcsclite ? null , pinentry ? null , readline ? null , sqlite ? null , zlib ?
+null
+}:
```

```suggestion
, adns, bzip2, gnutls, libusb1, openldap, tpm2-tss, pcsclite, pinentry, readline, sqlite, zlib
}:
```
there shouldn't be a space before ,